ci/pipelines/apps-test-ecr.yml

```diff
@@ -8,6 +8,14 @@ aws_config: &aws_config
   aws_ecr_registry_id: "((pay_aws_test_account_id))"
   aws_region: eu-west-1
 
+aws_staging_config: &aws_staging_config
```

Comment:
Should we rename the other block to `aws_test_config` for consistency?

Reply:
good idea, changed